components/payments/settings/payment-webhooks.tsx

```diff
@@ -29,16 +38,83 @@ export const PaymentsWebhooks: React.FC<PaymentsWebhooksProps> = ({
   const [testnetWebhooks, setTestnetWebhooks] = React.useState<PaymentsWebhooksType[]>([]);
 
   const { data: webhooks, isLoading: isGetLoading, isFetched, error } = usePaymentsWebhooksByAccountId(accountId);
+  const { mutate: updateWebhook, isLoading: isUpdateLoading, error: updateError, isSuccess: isUpdateSuccess } = usePaymentsUpdateWebhook(accountId);
+  const { mutate: createWebhook, isLoading: isCreateLoading, error: createError, isSuccess: isCreateSuccess } = usePaymentsCreateWebhook(accountId);
+
+  const [isUpdateDelete, setIsUpdateDelete] = React.useState(false);
+  const [currentDeleteWebhook, setCurrentDeleteWebhook] = React.useState<PaymentsWebhooksType>();
+  const [currentUpdateWebhook, setCurrentUpdateWebhook] = React.useState<PaymentsWebhooksType>();
+
+  const [currentCreateWebhook, setCurrentCreateWebhook] = React.useState<PaymentsWebhooksType>();
+
+  const [showAlert, setShowAlert] = React.useState(false);
+  const [alertHeader, setAlertHeader] = React.useState("");
```

Comment:
I believe you should need maximum one state here, you can write a record of error to header/description/status and then pick up the corresponding depending on the error gotten from the response call (check how I did it in KycStatus.tsx)

Also, you shouldn't really need useEffect, you should be able to use useMemo again here since the errors are already coming from the hooks.

In general we should depend on useEffect and useState veeeery rarely, having react query usually solves all our needs for it.